opennms-doc/guide-install/src/asciidoc/text/minion/troubleshooting.adoc

```diff
@@ -0,0 +1,51 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[gi-install-minion-troubleshooting]]
+=== Troubleshooting
+
+This section gives some hints how to troubleshoot _Minion_ deployments.
+The _{opennms-product-name}_ comes with a few use full built-in commands to verify configurations and debug access to _Management Agents_ through _Minion_.
+
+==== Verifying Connectivity
+
+Once the URLs and credentials for communicating with the _{opennms-product-name}_ instance are configured, you can verify connectivity using:
+
+.Verify connectivity with the _{opennms-product-name}_ controller
+[source]
+----
+admin@minion()> minion:ping
+Connecting to ReST...
+OK
+Connecting to Broker...
+OK
+admin@minion()>
+----
+
+==== Execute SNMP commands through a _Minion_
+
+The commands are available in the _Karaf CLI_ on the _{opennms-product-name}_ system.
+
+.Run SNMP commands from _{opennms-product-name}_ in a monitoring location
+[source]
+----
+ssh -oHostKeyAlgorithms=+ssh-dss -p 8101 localhost
+snmp:walk -l MyLocation IpAddressInMyLocation 1.3.6.1.4.1
+----
+
+This commmand allows you to verify the connection and _SNMP community_ configuration for a given host in a remote location.
+
+==== Run a monitor through a _Minion_
+
+.Show all available monitors
+[source]
+----
+poller:list-monitors
+----
+
+.Run an ICMP monitor through a _Minion_
+[source]
+----
+poller:poll -l MyLocation -t Time-To-Live-in-ms org.opennms.netmgt.poller.monitors.IcmpMonitor myIpAddress
```

Comment:
Specifying the TTL here, may raise some questions. Perhaps we should document that too, since it's important to understand when using ActiveMQ.